rfcs/0009-asset-registry-existential-deposit-workflow.md

```diff
@@ -0,0 +1,100 @@
+# Existential Deposit Values
+
+## Abstract
+---
+
+This document is a review of the implementation and usage of Existential Deposit
+values within our runtimes. This will review this maintainmance of ED values
+from a code and SUDO/Governance prospective.
+
+## Background
+---
+
+Existential Deposits are the minimum value required of a balance to exists on
+our chains. Given that different tokens will have different values, its 
+necessary to have different Existential Deposit values for different tokens.
+
+## Requirements
+---
+
+* Be able to define different Existential Deposit values for different
+  tokens.
+* Have a predefined collection of well-known tokens that each have their
```

Comment:
I think this is a technicality of the implementation to not have to define initial set, rather than a requirement, so we can possibly remove